lib/parameterized_test/formatter.ex

```diff
@@ -0,0 +1,61 @@
+defmodule ParameterizedTest.Formatter do
+  @moduledoc false
+  @behaviour Mix.Tasks.Format
+
+  @impl Mix.Tasks.Format
+  def features(_opts) do
+    sigil = if Version.match?(System.version(), "< 1.15.0"), do: :x, else: :PARAMS
```

Comment:
Would be nice if we could introspect the sigil and not have to duplicate this version check and sigil names here

Reply:
Agreed, but this seems fine!